go.mod

```diff
@@ -1,24 +1,26 @@
 module github.com/gardener/gardener-extension-os-ubuntu
 
-go 1.14
+go 1.15
 
 require (
-	github.com/gardener/gardener v1.10.0
+	github.com/gardener/gardener v1.12.5
```

Comment:
Hmm, where are the github templates?
Can you add a script like this https://github.com/gardener/gardener-extension-provider-aws/blob/master/hack/update-github-templates.sh?